docs/versioned_docs/version-zenith/quickstart/183917-daggerverse.mdx

```diff
@@ -0,0 +1,23 @@
+---
+slug: /quickstart/183917/daggerverse
+hide_table_of_contents: true
+title: "Use the Daggerverse"
+---
+import PartialExperimentalDocs from '../partials/_experimental.mdx';
+
+# Quickstart
+
+## Use the Daggerverse
+
+<PartialExperimentalDocs />
+
+As you've seen in previous sections, modules can be consumed from remote sources, or installed and used locally. The [Daggerverse](https://daggerverse.dev) is an online catalog of Dagger modules that enables users to easily discover and use modules created by the Dagger community.
+
+All Dagger CLI commands work as expected with Daggerverse URLs. Here are a few more examples for you to try:
+
+- [Publish a container to the ttlsh registry](https://daggerverse.dev/mod/github.com/shykes/daggerverse/ttlsh): `dagger call -m github.com/shykes/daggerverse/ttlsh publish --ctr alpine:latest`
+- [Play a game of yamlinvaders](https://daggerverse.dev/mod/github.com/sipsma/daggerverse/yamlinvaders): `dagger -m github.com/sipsma/daggerverse/yamlinvaders shell play`
```

Comment:
Just flagging that:
1. This would need to now be `dagger -m github.com/sipsma/daggerverse/yamlinvaders call play shell`
2. I'll need to update to use the new APIs for reading source files.
   * If I don't have time before we merge, can just rm it from this list for now

Reply:
Updated

Reply:
Updated the repo too to use the new APIs so it should work again in the new release